nss-sapphire-elephants/WorkforceManagement
# Issue ticket 15

Issue Ticket #15
Given the user wants to see past training programs
When the user is views the list of future training programs
Then the user should have an affordance labeled "View Past Programs" at the end of the list

Given the user is viewing the list of future training programs
When the user performs a gesture of the past programs affordance
Then the user should see a list of all training programs that ended before the current day

Also added a hyperlink to the Training Name in the main Index to link to details

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)

# Testing Instructions for Change Made
git fetch --all
git checkout issue-ticket-15
run

Can't test the hyperlink on name until the details link is finished but click on "View Past Programs" to see a list of past programs

# Checklist:

- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [x ] My changes generate no new warnings
- [x ] I have added test instructions that prove my fix is effective or that my feature works
